ophyd/signal.py

```diff
@@ -26,11 +26,23 @@ class Signal(OphydObject):
     timestamp : float, optional
         The timestamp associated with the initial value. Defaults to the
         current local time.
+    tolerance : any, optional
```

Comment:
There's nothing stopping you from using `set_and_wait` on an array PV, and with an appropriate tolerance `np.isclose` would handle it just fine, I think.